contracts/core/extensions/CoreAccounting.sol

```diff
@@ -208,22 +208,25 @@ contract CoreAccounting is
     )
         internal
     {
-        // Declare interface variavle for vault
-        IVault vault = IVault(state.vault);
+        // Don't allow withdraw if no amount
+        if (_quantity != 0) {
```

Comment:
Should this be `_quantity > 0` to mirror `depositInternal` and disallow negative quantities?

Reply:
Definitely should. Dunno why I did that.